server/schema/questions/index.js

```diff
@@ -1,12 +1,15 @@
 const ajv = require('../index')
 const createQuestionSchema = require('./createQuestion')
 const updateQuestionSchema = require('./updateQuestion')
+const addEditorSchema = require('./addEditor')
 
-ajv.addFormat('optionsPattern', (data) =>
+ajv.addFormat('arrPattern', (data) =>
   Array.isArray(JSON.parse(JSON.stringify(data)))
 )
 
 module.exports = {
   createQuestionSchema,
   updateQuestionSchema,
+  addEditorSchema,
+  removeEditorSchema: addEditorSchema,
```

Comment:
if doing this, might rename file to: `addOrRemoveEditor` and export as single entity

Reply:
then use the same for both routes

Reply:
How does that matter ? We are giving it a different name anyway

Reply:
Or it can also be `updateEditorSchema`, if that seems fine. Anyways, this will work fine too.